src/components/product/ProductCard/ProductCard.tsx

```diff
@@ -75,20 +78,22 @@ function ProductCard({
   return (
     <UICard className="product-card" data-card-variant={variant}>
       <UICardImage>
-        <Image
-          baseUrl={img.url}
-          sourceWidth={480}
-          aspectRatio={1}
-          width={360}
-          breakpoints={[250, 360, 480]}
-          layout="constrained"
-          backgroundColor="#f0f0f0"
-          options={{
-            fitIn: true,
-          }}
-          alt={img.alternateName}
-          sizes="(max-width: 768px) 200px, 320px"
-        />
+        <AspectRatio ratio={ratio}>
```

Comment:
Why do we need this component? Can't we use `aspect-ratio` css property?

Reply:
Basically, because the `aspect-ratio` property isn't fully supported yet. If you take a look at [Can I Use](https://caniuse.com/?search=aspect-ratio) results we can see that the property only has ~81% global coverage, which isn't interesting